.github/workflows/ci-testing-deploy.yml

```diff
@@ -79,7 +79,7 @@ jobs:
       anything-js: ${{ steps.filter.outputs.anything-js }}
     steps:
       - uses: actions/checkout@v3
-        if: ${{ github.event_name == 'push' }}
+        if: ${{ github.event_name != 'pull_request' }}
```

Comment:
@sanderegg how does this affect the rest? e.g. releases

Reply:
@pcrespov. so from the top of this file, this workflow ignores all tags, so I do not think this has any influence on releases.
actually we found that on pull_request event it seems the checkout is already done.
What I would be wondering @bisgaard-itis is whether that ```if``` is really necessary at all? Did you try to simply remove it?

Reply:
I just removed the `if`-statement. Let's see if the tests pass.